frontend/src/scenes/ingestion/v2/ingestionLogicV2.ts

```diff
@@ -251,22 +272,49 @@ export const ingestionLogicV2 = kea<ingestionLogicV2Type>([
                 setSidebarSteps: (_, { steps }) => steps,
             },
         ],
+        isDemoProject: [
+            teamLogic.values.currentTeam?.is_demo as null | boolean,
```

Comment:
> This doesn't work as if I'm currently on a non-demo team and go to /ingestion, I still have the option to create a new demo project

Are you sure? It works for me 😛 

The button is still there in case someone didn't realize that they had a demo project already, but when they click the button it just takes them to the demo project. There are also permissions on the API that should prevent the creation of multiple demo projects inside a single org. 

> I think you can just create a selector that reads from organizationLogic the current organization, and checks if any of the teams has is_demo.

I do that when checking if clicking on any of the "try with demo data" buttons should name a new demo project or simply redirect them there. https://github.com/PostHog/posthog/blob/b5b62fa572aeb119bf9177d6fae1f3f68563a834/frontend/src/scenes/ingestion/v2/panels/PanelComponents.tsx#L75

I think maybe you are misunderstanding what this is used for. `isDemoProject` is just used to check if the current project the user is viewing is a demo, and if so, take them to the screen that says "Sorry, you can't ingest events into a demo project."

LMK if that makes sense and if you still want to see changes here.